samples/openvx/no_wrappers.cpp

```diff
@@ -0,0 +1,429 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+
+//OpenCV includes
+//#include <opencv2/opencv.hpp>
+#include "opencv2/core.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+#include "opencv2/highgui.hpp"
+
+#ifndef VX_VERSION_1_1
+const vx_enum VX_IMAGE_FORMAT = VX_IMAGE_ATTRIBUTE_FORMAT;
+const vx_enum VX_IMAGE_WIDTH  = VX_IMAGE_ATTRIBUTE_WIDTH;
+const vx_enum VX_IMAGE_HEIGHT = VX_IMAGE_ATTRIBUTE_HEIGHT;
+const vx_enum VX_MEMORY_TYPE_HOST = VX_IMPORT_TYPE_HOST;
+const vx_enum VX_MEMORY_TYPE_NONE = VX_IMPORT_TYPE_NONE;
+const vx_enum VX_THRESHOLD_THRESHOLD_VALUE = VX_THRESHOLD_ATTRIBUTE_THRESHOLD_VALUE;
+typedef uintptr_t vx_map_id;
+#endif
+
+//simple wrappers around VX calls
+namespace ovx
+{
+vx_image createImage(vx_context context, vx_uint32 width, vx_uint32 height, vx_df_image format)
```

Comment:
while this sample doesn't use the OpenVX wrappers from '3rdparty' dir it still creates its own ones...
I expected it to use C-style OpenVX as is...
@vpisarev, do you think it would be better to remove these wrappers and start using OpenVX directly?